dotCMS/src/main/java/com/dotmarketing/factories/PersonalizedContentlet.java

```diff
@@ -9,7 +10,7 @@
  * @see com.google.common.collect.Table
  * @author jsanca
  */
-public class PersonalizedContentlet {
+public class PersonalizedContentlet implements Serializable{
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6315945)